src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -233,6 +234,7 @@ using LayerTypes = boost::variant<
     Linear<arma::mat, arma::mat, NoRegularizer>*,
     LinearNoBias<arma::mat, arma::mat, NoRegularizer>*,
     LogSoftMax<arma::mat, arma::mat>*,
+    Softmax<arma::mat, arma::mat>*,
```

Comment:
Other layers are in alphabetical order. Could you move the softmax layer accordingly?

Reply:
Thanks, @lozhnikov for the review. I've made the required changes. I noticed some other LayerTypes were also out of order, so arranged them as well.